lib/cocoapods/generate/installer.rb

```diff
@@ -37,7 +37,7 @@ def install_directory
       # @return [void]
       #
       def install!
-        UI.title "Generating #{spec.name} in #{UI.path install_directory}" do
+        UI.title "Generating workspace in #{UI.path install_directory}" do
```

Comment:
should this do `specs.count == 1 ? specs.first.name : 'workspace'`?

Reply:
actually, after reading the updated specs I think the current change in this PR is better and easier to read